### PR TITLE
Added CCPresetRevival.

### PR DESCRIPTION
**Pull request by keanuplayz**

### CODE REVIEW

File: mods.json [hunk 1]
```diff
@@ -1,5 +1,13 @@
 {
     "mods" : {
+        "ccpreset" : {
+            "name" : "Preset Revival",
+            "description" : "This brings back the preset menu, which allows you to start the game at a specific point.",
+            "page" : [{ "name" : "GitHub", "url" : "https://github.com/CCDirectLink/CCPresetRevival" }],
+            "archive_link" : "https://github.com/CCDirectLink/CCPresetRevival/archive/master.zip",
```
Comment (2767mr) [inline mods.json]: Should reference a release.

File: mods.json [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+            "hash" : { "sha256" : "b140da004846a75c859d0f44de1fff56d917fe43c8955f4d2564c79aa5ce84a8" },
```
Comment (2767mr) [inline mods.json]: Hash does not match since master has changed since PR was opened

File: mods.json [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+            "version" : "1.0.0"
```
Comment (2767mr) [inline mods.json]: Version does not match the `0.1.0` specified in it's `package.json`

File: mods.json (hunk 2 above)
Comment (keanuplayz) [reply mods.json]: Fixed.

**Pull request merged**